geopandas/io/sql.py

```diff
@@ -96,3 +96,222 @@ def load_geom_text(x):
                 crs = {"init": "epsg:{}".format(srid)}
 
     return GeoDataFrame(df, crs=crs, geometry=geom_col)
+
+
+def get_geometry_type(gdf):
```

Comment:
```suggestion
def _get_geometry_type(gdf):
```
I would keep these helper functions private.